src/haconiwa/core/applier.py

```
@@ -40,50 +42,195 @@ def apply(self, crd: Union[SpaceCRD, AgentCRD, TaskCRD, PathScanCRD, DatabaseCRD
                 return self._apply_database_crd(crd)
             elif isinstance(crd, CommandPolicyCRD):
                 return self._apply_commandpolicy_crd(crd)
+            elif isinstance(crd, OrganizationCRD):
+                return self._apply_organization_crd(crd)
             else:
                 raise CRDApplierError(f"Unknown CRD type: {type(crd)}")
         except Exception as e:
             logger.error(f"Failed to apply CRD {crd.metadata.name}: {e}")
             raise CRDApplierError(f"Failed to apply CRD {crd.metadata.name}: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)